datafusion/Cargo.toml

```diff
@@ -54,6 +54,10 @@ arrow = { version = "^5.3", features = ["prettyprint"] }
 parquet = { version = "^5.3", features = ["arrow"] }
 sqlparser = "0.11"
 paste = "^1.0"
+# these two are for approx_distinct
+twox-hash = "1.6.1"
```

Comment:
Is there any reason to use `XXHash` compared to `ahash` which is already a dependency: https://docs.rs/ahash/0.7.4/ahash/?